tests/integration-tests/tests/common/schedulers_common.py

```diff
@@ -141,7 +149,70 @@ def assert_job_succeeded(self, job_id, children_number=0):  # noqa: D102
         status = self.get_job_exit_status(job_id)
         assert_that(status).is_equal_to("0")
 
+    def compute_nodes_count(self):  # noqa: D102
+        result = self._remote_command_executor.run_remote_command("qhost | grep -o ip- | wc -l")
+        return int(result.stdout.split()[-1])
```

Comment:
Why do you need to split the stdout?

Reply:
To extract the last line and trim whitespaces:
```
>>> a = "a\n b"
>>> a.split()[-1]
'b'
```

Reply:
It is a very important reason.